### ISSUE TITLE
`com.hubitat.app.exception.LimitExceededException`

### ISSUE THREAD

**Issue by jessemcdowell** [claudegel/Hubitat-sinope-GT125#3]
I just upgraded my Hubitat yesterday to `2.3.4.132`. Since then, I've been seeing these kinds of errors in my Hubitat log for each of my Sinope thermostats:
`com.hubitat.app.exception.LimitExceededException: Device 12 generates excessive hub load on line 304 (method processChildResponse)`

![image](https://user-images.githubusercontent.com/384601/212564749-f26468cd-5ad2-4e59-b2d4-088d3bacf75c.png)

I'm using the following, installed by the Hubitat Package Manager:
* Sinope Neviweb Hub v1.2 (driver)
* Sinope Thermostat v1.2 (driver)

**Comment by rfg81** [comment]
Could you please try to reproduce it on latest? I made some improvements.

**Comment by jessemcdowell** [comment]
I upgraded my Hubitat to 2.3.4.153 over the weekend and I haven't gotten any of the reported errors or any other errors from this driver since.

Today I upgraded my Hubitat again to 2.3.5.101 (the latest) and upgraded Sinope Neviweb Hub GT125 to v1.4 via the Hubitat Package Manager. I still don't see any errors.

I'm afraid I can't tell you if your fix helped because it seems something in the Hubitat corrected the errors first. Either way, I think this issue can be closed.

Thanks for the help!
